packaging/Dockerfile.centos_7_git_rpm

```diff
@@ -0,0 +1,20 @@
+# This Dockerfile is based on the recommendations provided in the
+# Centos official repository (https://hub.docker.com/_/centos/).
+# It enables systemd to be operational.
+FROM centos:7.5.1804
```

Comment:
This one should be based on the previous image.
`FROM centos7-nmstate-base`
And add only the additions here.